src/shared.js

```diff
@@ -150,8 +150,9 @@ export function foldLine (str, maxLen = 76) {
 /**
  * Comparator function for comparing msgid
  *
- * @param {{msgid: string}} left with msgid prev
- * @param {{msgid: string}} right with msgid next
+ * @template {Buffer|string} T
+ * @param {{msgid: T}} left with msgid prev
+ * @param {{msgid: T}} right with msgid next
```

Comment:
This one actually is used on both strings and buffer, though only to compare a single type at a time.